vendor/engines/secure_rooms/app/models/secure_rooms/indala_ability.rb

```diff
@@ -0,0 +1,23 @@
+module SecureRooms
+
+  class IndalaAbility
+
+    include CanCan::Ability
+
+    def initialize(user, facility)
+      return unless user && facility
+
+      if user.administrator? || user_has_role_at_facility?(user, facility)
+        can :edit, User
```

Comment:
This can be shortened to `can [:edit, :update], User`. And I thought that CanCanCan aliased the actions, so can we get away with only one or the other?